### ISSUE TITLE
GrpclbLoadBalancer should have full control over the lifecycle of StopWatch

### ISSUE THREAD

**Issue by dapengzhang0** [grpc/grpc-java#5534]
Would be good to add another `GrpclbLoadBalancer` constructor without the `StopWatch` argument. The old constructor is just `@VisibleForTesting`, and `GrpclbLoadBalancerProvider` will call the new constructor. In this way, `GrpclbLoadBalancer` can have full control over the lifecycle of the `StopWatch`.

**Comment by waveywaves** [comment]
I would like to start working on this. Anything I need to keep in mind before I start ?

**Comment by dapengzhang0** [comment]
Thank you for working on this. Nothing special to keep in mind. If you are first-time contributing grpc-java, just not to break the [guideline](https://github.com/grpc/grpc-java/blob/master/CONTRIBUTING.md)

**Comment by waveywaves** [comment]
@dapengzhang0 Yes I am a first time contributer to this repo. Thank you for that. Can you assign this to me ?

**Comment by dapengzhang0** [comment]
I can not assign this to you. Seems you are not in certain user group that is assignable for grpc-java issues.
